CHANGELOG.md

```diff
@@ -11,6 +11,17 @@
 * None.  
 
 
+## 1.10.0.rc.2 (2020-10-13)
```

Comment:
is this needed? we have not released 1.10.0.rc.2 yet. If you want you can target the `1-10-stable` branch for this change so it can ship with 1.10.